index.html

```diff
@@ -1,228 +1,188 @@
-                <a href="#">airbnd</a>
-                <a href="#">daumkakao</a>
-
+            <div class="main-desc">
+              <p>
+                Now shifting to other countries with your family isn’t a
+                problem.
+              </p>
             </div>
-        </section>
-        <section class="footer">
-            <div class="footer-sec">
-                <div><img src="images/logo.png" alt="" class="footer-logo" /></div>
-                <div class="footer-info">BAKHTAR <br>MIGRATION INC</div>
-
+          </div>
+          <div class="main-div">
+            <div class="lecturer">
+              <div class="main-img">
+                <img class="logo" src="images/s6.png" alt="" />
+              </div>
+              <div class="main-title">Work Permit</div>
             </div>
-            <div class="right">
-                
-                <p>2022 Migration to Australia>All rights reserved<br>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet perspiciatis vitae quo quisquam
-                    debitis a dignissimos fugit. Aliquam dolorum possimus, </p>
+            <div class="main-desc">
+              <p>
+                You will normally need to be in one of the following situations
+                to have a working visa in your destination Country
+              </p>
             </div>
-        </section>
+          </div>
+        </div>
+
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+        <div class="main-sec">
+          <h1>Join Bakhtar for consultancy services</h1>
+        </div>
+      </section>
+
+      <!-- speakers container-->
+      <section id="speakerContainer" class="speakers">
+        <h1>Featured Speakers</h1>
+        <div id="speakers-container">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+
+        <div class="moreSpeakers">
+          <a href="#">more</a>
+        </div>
+      </section>
+
+      <section class="partner1" id="join-section">
+        <div class="partner1-img">
+          <h1>partner</h1>
+
+          <a href="#">mazila</a><a href="#">Google</a><a href="#">NAVER</a>
+          <a href="#">airbnd</a>
+          <a href="#">daumkakao</a>
+        </div>
+      </section>
+      <section class="footer" id="news-section">
+        <h2 class="fo">footer</h2>
+        <div class="footer-sec">
+          <div><img src="images/logo.png" alt="" class="footer-logo" /></div>
+          <div class="footer-info">BAKHTAR <br />MIGRATION INC</div>
+        </div>
+        <div class="right">
+          <p>BAKHTAR Migration INC.some rights reserved</p>
+        <p class="p2">
+           Address: No. 15,Barchee St.,
+          13 District, Kabul City , Afghanistan, If you want to contact us for
+          official business, your call will be transferred to the appropriate
+          department.
+        </p>
+        </div>
+      </section>
```

Comment:
in the footer section of you page, there should be more margin around the contents of the footer and the body `justify-content: center` could if fix this and more padding between the logo and the address

your design 
![image](https://user-images.githubusercontent.com/60612329/191740766-5368db13-d05b-4fb8-ad5c-600e2bab5268.png)

required design
![image](https://user-images.githubusercontent.com/60612329/191741249-a4a251b8-eeb5-4108-b905-b87427eaf524.png)